docs/pages/components/tile.md

```diff
@@ -10,186 +10,911 @@ summary:
-    <div class="fd-product-tile__content">
-        <h3 class="fd-product-tile__title">Disabled Product Tile</h3>
+## Slide Tile
+The Slide Tile contains a news headline, a background image, the news source, and a time stamp. A start and pause button are always displayed on touch devices and can be revealed by hovering the mouse over the tile on desktop devices. The dots on the bottom of each tile identify the number of pages they contain. 
+To create a Slide Tile, use the `fd-tile--slide` modifier class.
+
+{% capture tile-slide %}
+<div class="docs-section-container">
+    <div tabindex="0" class="fd-tile fd-tile--slide">
+        <div class="fd-tile__background-img" style="background-image: url('https://placeimg.com/200/200/nature')"></div>
+        <button aria-label="toggle play button" class="fd-tile__toggle"></button>
+        <div class="fd-tile__container">
+            <div class="fd-tile__header">
+                <div class="fd-tile__title">Wind Map: Monitoring Real-Time</div>
+                <div class="fd-tile__subtitle">SAP News</div>
+            </div>
+            <div class="fd-tile__footer">
+                <span class="fd-tile__footer-text">May 14, 2020</span>
+            </div>
+            <div class="fd-tile__page-indicator">
+                <span class="fd-tile__dot"></span>
+                <span class="fd-tile__dot"></span>
+                <span class="fd-tile__dot fd-tile__dot--active"></span>
+                <span class="fd-tile__dot"></span>
+                <span class="fd-tile__dot"></span>
+            </div>
+        </div>
+    </div>
+    <div tabindex="0" class="fd-tile fd-tile--slide fd-tile--s">
+        <div class="fd-tile__background-img" style="background-image: url('https://placeimg.com/200/200/nature')"></div>
+        <button aria-label="toggle play button" class="fd-tile__toggle"></button>
+        <div class="fd-tile__container">
+            <div class="fd-tile__header">
+                <div class="fd-tile__title">Wind Map: Monitoring Real-Time</div>
+                <div class="fd-tile__subtitle">SAP News</div>
+            </div>
+            <div class="fd-tile__footer">
+                <span class="fd-tile__footer-text">May 14, 2020</span>
+            </div>
+            <div class="fd-tile__page-indicator">
+                <span class="fd-tile__dot"></span>
+                <span class="fd-tile__dot"></span>
+                <span class="fd-tile__dot fd-tile__dot--active"></span>
+                <span class="fd-tile__dot"></span>
+                <span class="fd-tile__dot"></span>
+            </div>
+        </div>
+    </div>
+</div>
+<div class="docs-section-container">
+    <div tabindex="0" class="fd-tile fd-tile--double fd-tile--slide">
+        <div class="fd-tile__background-img" style="background-image: url('https://placeimg.com/400/200/nature')"></div>
+        <<button aria-label="toggle pause button" class="fd-tile__toggle fd-tile__toggle--pause"></button>
```

Comment:
Extra `<` causing:
![image](https://user-images.githubusercontent.com/43764832/82719296-14520580-9c5e-11ea-8219-2b810d5a1ad5.png)

Reply:
Thanks for catching this. I will fix it

Reply:
https://github.com/SAP/fundamental-styles/pull/1054